.github/workflows/run_pr_tests.yml

```diff
@@ -194,3 +194,43 @@ jobs:
         hal_refsi_soc: G1
         hal_refsi_thread_mode: WG
         debug_support: ON
+
+  # Based on: mr-ubuntu-clang-x86-llvm-previous-cl3-0-offline
+  run-ubuntu-clang-x86-llvm-latest-cl3-0-offline:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 90 # offline needs longer timeout
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4.1.0
+    - name: setup-ubuntu
+      uses: ./.github/actions/setup_build
+      with:
+        llvm_version: '19'
+        llvm_build_type: RelAssert
+        arch: x86
+    - run: echo WORKSPACE is $GITHUB_WORKSPACE && echo PWD is `pwd` && ls -al
+    - name: build ock x86 relassert
+      uses: ./.github/actions/do_build_ock
+      with:
+        build_32_bit: ON
+        c_compiler: $GITHUB_WORKSPACE/llvm_install/bin/clang
+        cxx_compiler: $GITHUB_WORKSPACE/llvm_install/bin/clang++
+        build_targets: check-ock
+        enable_api: ""
```

Comment:
@coldav Do you want this to be the default across the board? 
Currently about 67% of the do_build_ock calls use "cl".

Reply:
keep the old behaviour for the old tests, I think they will get dropped in time as you move things across anyway.